src/analytics/src/main/scala/osmesa/analytics/oneoffs/ChangesetStatsCreator.scala

```diff
@@ -155,6 +178,16 @@ object ChangesetStatsCreator
                 }
               })
 
+            // Distributing these writes to the executors to avoid no suitable driver errors on master node
+            logger.warn(s"Writing AugmentedDiffStream sequence number as $augdiffEndSequence to $databaseUrl")
+            spark.sparkContext.parallelize(Seq(databaseUrl)).foreach { uri =>
+              ChangesetORCUpdaterUtils.saveLocations("AugmentedDiffStream", augdiffEndSequence, uri)
+            }
+            logger.warn(s"Writing ChangesetStream sequence number as $changesetsEndSequence to $databaseUrl")
+            spark.sparkContext.parallelize(Seq(databaseUrl)).foreach { uri =>
+              ChangesetORCUpdaterUtils.saveLocations("ChangesetStream", changesetsEndSequence, uri)
+            }
```

Comment:
Ah, these are the calls I was I was aspiring to in https://github.com/azavea/osmesa/pull/157/files#r311292846